lib/helpers/index.js

```diff
@@ -2,5 +2,26 @@ const localAssets = require('./local-assets');
 const packageURL = require('./package-url');
 const getDefaults = require('./get-defaults');
 const configStore = require('./config-store');
+const typeSlug = require('./type-slug');
+const typeTitle = require('./type-title');
+const pathDiff = require('./path-diff');
+const {
+    addTrailingSlash,
+    removeTrailingSlash,
+    addLeadingSlash,
+    removeLeadingSlash,
+} = require('./path-slashes');
```

Comment:
With ESM modules not supporting the index.js pattern would it make sense to avoid using it so that migrating to ESM modules is easier in the future?

Reply:
I don't see it as a huge deal I don't think, you can still do

```js
import pathSlashes from './path-slashes';
const { addLeadingSlash } = pathSlashes;
```

Most likely I would migrate the path slashes module itself to esm and it would work on one line. The alternative is that I would need to have a file for every function which would result in multiple import statements right? Anything I'm missing in my thinking here?

Reply:
Sorry I meant when loading importing not when exporting from this file. With ESM you would have to do `import {addLeadingSlash} from "../helpers/index.js"` might as well go `import {addLeadingSlash} from "../helpers/path-slashes.js"`?

Reply:
ah true. I should probably use package exports to set up the various helpers and such the common package exposes

Reply:
I would not worry too much about this at the moment. I would like a separate task on reworking the public API of this module. The current public API is a bit confusing (probably due to a lot of the code being moved here from the internals of the client). 

Ex: to load the eik config in the plugins we're now calling `helpers.getDefaults(path)`. That's not really a well described method for loading a config.